Docs 8010 edit api groups pa (#255)
mulesoft/docs-api-manager: modules/ROOT/pages/api-groups-landing-page.adoc

> = API Groups

Reviewer: Superfast proof/scan review

Author: Ok Thanks!

> = API Groups

Reviewer: ```suggestion
= Advantages of Using API Groups
```

Author: Hmm...this page is that and above so will keep as is for now.

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> API Groups enables you to bundle your APIs and specific resources to solve specific user needs. Instead of individually using a list of resources and APIs, the API users can request access to and use these APIs as a package that solves a specific problem for them. 

Reviewer: ```suggestion
Using API Groups enables you to bundle your APIs and resources to solve specific user needs. Instead of using individual resources and APIs from a list, the users can request access to and use these in a package that solves a specific problem for them. 
```

Author: Done! Thanks!

> When you create an API Group, you can create different instances of that group with different SLAs and rate limit conditions based on the problem it must solve. You can then publish the API Group to Anypoint Exchange for users to be able to subscribe to use the package.

Reviewer: ```suggestion
When you create an API Group, you can customize instances of that group with different SLAs and rate limit conditions based on the problem the instance must solve. You can then publish the API Group to Anypoint Exchange so that users can subscribe to the package.
```

Author: Done! Thanks!

Because each API Group can include several API instances, and each instance can be included in multiple API Group solutions, package customization options are limitless. 

> == How API Groups Works

Reviewer: ```suggestion
== API Groups Architecture
```

Author: Done! Thanks!

Author: Changed it to How APU Groups Works.

> As an API Group Administrator, you create an API Group for a specific organization and environment. API Groups are versioned, like the APIs, to easily maintain changes.  

Reviewer: ```suggestion
As an API Group Administrator, you create an API Group for a specific organization and environment. API Groups are versioned, like the APIs contained within them, to more easily incorporate updates.  
```

Author: Done! Thanks!

> You can create instances of an API Group in the same or different environments and add different SLA and rate limit conditions. 

Reviewer: Haven't we said this already?

Author: Removed it! Done! Thanks!

> The following diagram explains how you can implement API Groups in different environments.

Reviewer: ```suggestion
The following graphic illustrates how you can implement API Groups in different environments.
```

Author: Done! Thanks!

image:api-groups-hierarchy.png[]

> == Use Case

Reviewer: ```suggestion
== Example Issue to Solve Using API Groups
```

Author: Done! Thanks!

> To better understand how API Groups facilitates API consumer engagement, let us look at the example of the Good Weather company. Good Weather is a weather forecasting company that  predicts the atmospheric conditions for a given location and time. 

Reviewer: ```suggestion
To better understand how API Groups facilitates API consumer engagement, consider the needs of a fictitious company named Good Weather, which forecasts atmospheric conditions for a given location and time by using meteorology. 
```

Author: Done! Thanks!

> Good Weather collects quantitative data about the current state of the atmosphere at a given place and using meteorology. Good Weather enables consumers to access their weather forecast data using the following APIs:

Reviewer: ```suggestion
Good Weather enables customers to access requested results using the following APIs:
```

Author: Done! Thanks!

* Locations API
* Forecast API
* Current Conditions API
* Indices API
* Weather Alarms API
* Alerts API
* Imagery API
* Tropical API
* Translations API

> Currently, the consumers access and use the APIs by:

Reviewer: ```suggestion
Currently, the process that customers must follow to access and use the APIs is not efficient, requiring:
```

Author: Done! Thanks!

> * Scanning the Good Weather company portal to find appropriate APIs to actualize a business flow. The consumers have to determine the API they need by reviewing a long list of APIs on the portal. Additionally, the problem is compounded when developers modify the existing APIs or introduce additional APIs. 

Reviewer: GLOBALLY fix each bullet item to use this format:
```suggestion
* Searching the Good Weather company portal to find appropriate APIs to actualize a business flow +
To determine the precise API they need, customers have to search a long list of APIs on the portal. This difficulty is compounded when developers modify the existing APIs or introduce additional APIs. 
```

Author: Done! Thanks!

> * Changing multiple APIs to adopt a change to a service plan for each API. This complicates the task of upgrading or downgrading of services.

Reviewer: ```suggestion
* Changing multiple API service plans individually to adopt a service-wide change 
This upgrading or downgrading services.
```

Author: Done! Thanks!

> * Dealing with access to all the elements of an API when an API access is approved. The API packaging does not restrict access down to the individual call and data elements. This introduces security issues and unnecessary overheads.

Reviewer: ```suggestion
* Working to access each element of an API when API access is approved
The API packaging does not restrict access down to the individual call and data elements. This introduces security issues and unnecessary overhead.
```

Author: Done! Thanks!

> == How API Groups Derives Value for Good Weather

Reviewer: ```suggestion
== API Groups Solution
```

Author: Done! Thanks!

> Using API Groups, Good Weather can package different APIs based on consumer requirements and scenarios, with specific SLAs that do not clash with the requirements for other consumers. Good Weather can quickly adapt and absorb API changes. With the API Groups: 

Reviewer: ```suggestion
Using API Groups, Good Weather can package different APIs based on consumer requirements and scenarios, with specific SLAs that do not clash with the requirements for other consumers. Good Weather can quickly adapt and absorb API changes. Using API Groups: 
```

Author: Done! Thanks!

> * Good Weather can package different sets of APIs to create a business model targeted to different consumers as per their requirements.

Reviewer: ```suggestion
* Good Weather can package different sets of APIs to target different consumers' requirements.
```

Author: Done! Thanks!

> * API consumers can request access to a collection of related APIs instead of individual APIs, which simplifies discovery of assets on the Good Weather portal. 

Reviewer: ```suggestion
* API consumers can access a collection of related APIs instead of individual APIs, simplifying asset discovery on the Good Weather portal. 
```

Author: Done! Thanks!

> * API Groups Governance can be enforced using SLA plans, which can be modified with a single click on an API Group.

Reviewer: ```suggestion
* API Groups can enforce governance using SLA plans that can be modified with a single click.
```

Author: Done! Thanks!

> The following screenshot illustrates the API Group packages that Good Weather created to meet their business requirements.

Reviewer: ```suggestion
The following screenshot illustrates the API Group packages that Good Weather created to meet their business requirements:
```

Author: Done! Thanks!

//ask Meeta about legal implications of adding this screenshot.

image::api-product-packages.png[API Groups Packages for Weather Forecasting]

== See Also

* xref:api-groups-landing-page.adoc[API Groups]
* xref:api-groups-creating-groups.adoc[Create Groups]
* xref:api-groups-modifying-groups.adoc[Modify Groups]
* xref:api-groups-publishing-to-exchange.adoc[Publishing API Groups to Exchange]
* xref:api-groups-deprecating-groups.adoc[Deprecate Groups]
* xref:api-groups-deleting-groups.adoc[Deleting API Groups]

>  

Reviewer: BREAK HERE

Reviewer: RESUME
